modules/registry-configuring-storage-baremetal.adoc

```diff
@@ -41,8 +69,9 @@ $ oc get pod -n openshift-image-registry
 [NOTE]
 =====
 * If the storage type is `emptyDIR`, the replica number cannot be greater than `1`.
-* If the storage type is `NFS`, you must enable the `no_wdelay` and `root_squash` mount options. For example:
-+
+* If the storage type is `NFS`, and you want to scale up the registry Pod by setting `replica>1` you must enable the `no_wdelay` mount option. For example:
```

Comment:
We removed NFS examples in image registry docs. See these lines as example: https://github.com/openshift/openshift-docs/pull/26512/files#diff-4af87df61050e4b8fe830bbb6be46aaab303d94d8e0b79e1ede86f612413395dL72-L103